lib/elixir_console_web/live/console_live.html.leex

```diff
@@ -1,3 +1,5 @@
+<% alias ElixirConsoleWeb.ConsoleLive.{CommandInputComponent, HistoryComponent, SidebarComponent} %>
```

Comment:
I feel this line should be in the `ConsoleLive` module (it works, I've checked). So we can have all the helper functions and imported things there, and the template is only interpolated HTML.

I bet this is more natural, the "context" is taken from the corresponding Elixir module.

Reply:
Yes! That's how I wanted it to work but I thought and pretty sure I tested it and couldn't make it work, probably I was doing something wrong because now it works and it's much better than having those aliases